packages/augur-sdk/src/state/db/DB.ts

```diff
@@ -167,36 +167,45 @@ export class DB {
     for (let trackedUser of await this.trackedUsers.getUsers()) {
       for (let userSpecificEvent of this.userSpecificEvents) {
         let dbName = this.getDatabaseName(userSpecificEvent.name, trackedUser);
+        try {
+          dbSyncPromises.push(
+            this.syncableDatabases[dbName].sync(
+              augur,
+              chunkSize,
+              blockstreamDelay,
+              highestAvailableBlockNumber
+            ));
+        } catch (e) {
```

Comment:
We should at least be logging the errors we're eating.

Reply:
kk, I'll change that.